saskatoon/member/views.py

```diff
@@ -85,6 +85,19 @@ def get_form_kwargs(self, *args, **kwargs):
         return kwargs
 
 
+class OnboardingPersonUpdateView(PersonUpdateView):
+    form_class = OnboardingPersonUpdateForm
+
+    def has_permission(self):
+        # Allow person to update own person
+        if self.get_object().pk == self.request.user.person.pk:
+            return True
+        return super().has_permission()
```

Comment:
what are the permissions on this view? Is it public?
I see two options here:
1. Use `LoginRequiredMixin` and simply set :
```
    def has_permission(self):
        return self.get_object().pk == self.request.user.person.pk
```

2. Use PermissionRequiredMixin with `permission_required` = `member.change_person`
but I don't really see what would be the advantage of that

Reply:
went with `LoginRequiredMixin` since second allows to change any person